pkg/types/installconfig.go

```diff
@@ -351,3 +360,33 @@ type BootstrapInPlace struct {
 	// InstallationDisk is the target disk drive for coreos-installer
 	InstallationDisk string `json:"installationDisk"`
 }
+
+// WorkloadPartitionName is the name of the workload partition
+//
+// The first release only supports a single workload partition whose name must
+// be "management".
+// +kubebuilder:validation:Enum="";management
+type WorkloadPartitionName string
+
+const (
+	// ManagementWorkloadPartition pins all infrastructure pods to the
+	// defined cpuset.
+	ManagementWorkloadPartition WorkloadPartitionName = "management"
```

Comment:
That's an interesting thing... The explicit value `management` is what the name should be because then it will really match what the crio and kubelet configs get.  So do we really want the user input to not match what ends up in the configs?

Reply:
That makes sense. This isn't really an enum but rather a name with limited values.